Final Merge

vjspranav opened the pull request:
A working simulation final merge into main

pavanchow committed:
Merge pull request #2 from virtual-labs/dev

Dev

vjspranav committed:
Initial orifice implementation
* Animation Works
* Buttons Work

PanwarJayant committed:
Merge pull request #3 from virtual-labs/dev

Initial test

PanwarJayant committed:
Update main.js

PanwarJayant committed:
Update main.js

PanwarJayant committed:
Update main.js

csart03 committed:
responsiveness increased

csart03 committed:
Merge branch 'testing' into dev

csart03 committed:
Merge pull request #4 from virtual-labs/dev

Dev

vjspranav committed:
Make animation Speed dynamic

vjspranav committed:
Merge pull request #5 from virtual-labs/dev

Make animation Speed dynamic

csart03 committed:
responsiveness increased

csart03 committed:
Merge branch 'dev' of https://github.com/virtual-labs/exp-orifices-iiith into dev

csart03 committed:
Merge pull request #6 from virtual-labs/dev

Dev

PanwarJayant committed:
Update procedure.md

PanwarJayant committed:
Update theory.md

PanwarJayant committed:
Merge pull request #7 from virtual-labs/dev

Dev

sravanthimodepu committed:
added assignment.md file

Commit:
Remove dublication of assignment.md file

balamma committed:
renamed Observations.md in to observations.md

balamma committed:
Merge branch 'dev' of https://github.com/virtual-labs/exp-orifices-iiith into testing

balamma committed:
added pre and posttest with sample questions

balamma committed:
Merge branch 'dev' of https://github.com/virtual-labs/exp-orifices-iiith into testing

balamma committed:
added pre and post tests .json formates and remvoved pre and post tests .js formates

balamma committed:
Merge branch 'dev' of https://github.com/virtual-labs/exp-orifices-iiith into testing

vjspranav committed:
Improvide variable names
* Add Proper heights
* Include our script the last

vjspranav committed:
Merge pull request #8 from virtual-labs/dev

Improvide variable names

raj-vlabs reviewed (changes requested):
1. I am not able to see some of the files like aim.md, experiment-name.md, objectives.md, references.md etc. Please ensure all files are present in the repo.
2. Pretest and posttest have sample questions. Please remove
3. .vscode/settings.json need not be checked in. If there's a valid use-case, please let me know.
4. Can we use better names than startAn and unln please?

PanwarJayant committed:
Update pretest.json

PanwarJayant committed:
Update posttest.json

PanwarJayant committed:
Merge pull request #10 from virtual-labs/dev

Dev

PanwarJayant committed:
Update pretest.json

PanwarJayant committed:
Update posttest.json

vjspranav committed:
Merge pull request #11 from virtual-labs/dev

Dev

vjspranav committed:
Change function names

vjspranav committed:
Merge pull request #12 from virtual-labs/dev

Change function names

vjspranav commented:
> 1. I am not able to see some of the files like aim.md, experiment-name.md, objectives.md, references.md etc. Please ensure all files are present in the repo.
> 2. Pretest and posttest have sample questions. Please remove
> 3. .vscode/settings.json need not be checked in. If there's a valid use-case, please let me know.
> 4. Can we use better names than startAn and unln please?

1. The files mentioned are already present in main branch
2. Added question in pre-test and post-test
3. Removed the file sir
4. Updated function names

raj-vlabs commented on experiment/simulation/js/main.js:
```diff
@@ -1 +1,83 @@
+    path.setAttribute("offset", 0);
+  });
+};
+disablestart = false;
```
Where's this variable declared and where this is used?

raj-vlabs commented on experiment/simulation/js/main.js:
```diff
@@ -1 +1,83 @@
+  });
+};
+
+ids = ["complete-flow"];
```
No declaration for the variable.

raj-vlabs reviewed (changes requested):
1. Missing declaration for variable 'ids'.
2. Unused variable 'disablestart'.

vjspranav committed:
Don't use undeclared variables
* Remove unused variable that became redundant after switching button disable to bootstrap css

vjspranav committed:
Merge pull request #13 from virtual-labs/dev

Don't use undeclared variables
